extensions/sql-migration/src/wizard/migrationModePage.ts

```diff
@@ -0,0 +1,78 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import { MigrationWizardPage } from '../models/migrationWizardPage';
+import { MigrationCutover, MigrationStateModel, StateChangeEvent } from '../models/stateMachine';
+import * as constants from '../constants/strings';
+
+export class MigrationModePage extends MigrationWizardPage {
+	constructor(wizard: azdata.window.Wizard, migrationStateModel: MigrationStateModel) {
+		super(wizard, azdata.window.createWizardPage(constants.DATABASE_BACKUP_MIGRATION_MODE_LABEL), migrationStateModel);
+		this.wizardPage.description = constants.DATABASE_BACKUP_MIGRATION_MODE_DESCRIPTION;
+	}
+
+	protected async registerContent(view: azdata.ModelView): Promise<void> {
+		const form = view.modelBuilder.formContainer()
+			.withFormItems(
+				[
+					this.migrationModeContainer(view),
+				]
+			);
+		await view.initializeModel(form.component());
+	}
+
+	public async onPageEnter(): Promise<void> {
+	}
+	public async onPageLeave(): Promise<void> {
+	}
+	protected async handleStateChange(e: StateChangeEvent): Promise<void> {
+	}
+
+	private migrationModeContainer(view: azdata.ModelView): azdata.FormComponent {
+		const buttonGroup = 'cutoverContainer';
+
+		const onlineButton = view.modelBuilder.radioButton().withProps({
+			label: constants.DATABASE_BACKUP_MIGRATION_MODE_ONLINE_LABEL,
+			name: buttonGroup,
+			checked: true
+		}).component();
+
+		this.migrationStateModel._databaseBackup.migrationCutover = MigrationCutover.ONLINE;
+
+		onlineButton.onDidChangeCheckedState((e) => {
```

Comment:
All of these event handlers should be added to a disposable store and disposed of properly when the wizard is closed (or whenever they're not needed anymore)

Reply:
Will do this in a future PR.